### ISSUE TITLE
update favicon and twitter metatags

### ISSUE THREAD

**Issue by owocki** [base0010/gitcoin-dance#3]
https://share.getcloudapp.com/2Nuq7q7O

**Comment by dylanavery720** [comment]
fixed and pushed ... however the favicon im using is probably not the right one, i just pulled a small svg from the gitcoin logo asset folder at /press ... ill probably need a better favicon

**Comment by owocki** [comment]
can we invite melvin here + tag him on design tasks?

**Comment by dylanavery720** [comment]
yeah for sure, i think @base0010 will have to handle that in settings though
